src/wazuh_db/wdb.h

```diff
@@ -476,9 +481,30 @@ int wdb_create_profile(const char *path);
 /* Create new database file from SQL script */
 int wdb_create_file(const char *path, const char *source);
 
-/* Get an array containing the ID of every agent (except 0), ended with -1 */
+/**
+ * @brief Returns an array containing the ID of every agent (except 0), ended with -1.
+ * This method creates and sends a command to WazuhDB to receive every the ID of every agent.
```

Comment:
Reword this sentence. `to receive every the ID of every agent`

Reply:
Fixed